unit_tests/falco/test_configuration.cpp

```diff
@@ -174,3 +174,34 @@ TEST(Configuration, configuration_environment_variables)
     /* Clear the set environment variable after testing */
     unsetenv(env_var_name.c_str());
 }
+
+TEST(Configuration, configuration_webserver_ip)
```

Comment:
Since we are now accepting user input, lets also check for some typos such as accidental white spaces and such for completeness? WDYT?

Reply:
Sure thing